src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from '../styles/Header.module.css';
+
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+    color: '#aeadad',
+    textAlign: 'center',
+  };
+  return (
+    <header style={headerStyle} className={styles.header}>
```

Comment:
It would be better to add just one kind of style ( object or from a CSS file ), just as you did both, it should be one.
It will make another person easier to edit these files when needed. The programmer can just change values from one place without the need to go to 2 different locations 🤪